### PR TITLE
update readme: fallback cta icon is star

### PR DESCRIPTION
**Pull request by oshi97**

### CODE REVIEW

**Comment by oshi97** [comment]
from this thread! https://yext.slack.com/archives/GREFVUD7V/p1587420183160800

File: README.md [hunk 1]
```diff
@@ -652,7 +652,8 @@ Note: A CTA without both a label and icon will not be rendered.
 const callsToAction = [{
   // Label below the CTA icon, default null
   label: 'cta label',
-  // Icon name for the CTA that is one of the SDK icons, default to undefined (no icon)
+  // Icon name for the CTA that is one of the SDK icons, defaults to undefined (no icon). If your icon
+  // is not recognized it will default to 'star'.
```
Comment (alexisgrow) [inline README.md:656]: Is this what happens? That seems fancy/magical and we don't do that anywhere else for icons, right?

Comment (oshi97) [reply README.md:656]: Oh no we do this everywhere for icons!

Comment (alexisgrow) [reply README.md:656]: woah, TIL! Awesome, LGTM!

**Review by alexisgrow — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (oshi97) [reply README.md:656]: It's in lines 146-149 of handlebarsrenderer.js

**Pull request merged**